src/components/Topics.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { getTopics } from "../api";
+import TopicList from "./TopicList";
+import { useEffect } from "react";
+const Topics = () => {
+    const [topics,setTopics] = useState([])
+    const [err,setErr]=useState(null)
+
+   
+
+   useEffect(()=>{
+    getTopics()
+    .then((topicData)=>{
+        const topicCalledAllIncl= [{slug: 'All articles', description: 'Choose from any topic'}, ...topicData]
+     
+        setTopics(topicCalledAllIncl)
+  
+   })
+   .catch((err)=>{
+    setErr(err)
+   })
+   },[])
+    return ( 
+ 
+        <div>
+            <h2>Please select a topic/view all articles</h2>
+            <TopicList topics={topics}/>
+        </div>
+     );
+}
```

Comment:
This is really good, but same as the above. This is a feature outside the scope of simply rendering all articles